clusters/app.ci/assets/dptp-controller-manager.yaml

```diff
@@ -67,6 +67,14 @@ rules:
   - "roles"
   verbs:
   - "*"
+- apiGroups:
+  - testplatform.x-openshift.io
```

Comment:
We've used `ci.openshift.io` in the past for e.g. labels -- anything wrong with that namespace?

Reply:
Nope, done